campaign-launcher/interfaceV2/src/components/modals/CreateCampaignModal/index.tsx

```diff
@@ -122,276 +154,340 @@ const CreateCampaignModal: FC<Props> = ({ open, onClose }) => {
+                      <FormHelperText>{errors.exchange.message}</FormHelperText>
+                    )}
+                  </FormControl>
+                  <InfoTooltip />
+                </Box>
+                <FormControl error={!!errors.pair} sx={{ width: '100%' }}>
+                  <Controller
+                    name="pair"
+                    control={control}
+                    render={({ field }) => {
+                      return (
+                        <Autocomplete
+                          id="trading-pair-select"
+                          options={tradingPairs || []}
+                          slotProps={slotProps}
+                          renderInput={(params) => (
+                            <TextField {...params} label="Trading Pair" />
+                          )}
+                          renderOption={(props, option) => {
+                            return (
+                              <Box
+                                {...props}
+                                key={option}
+                                component="li"
+                                sx={{ '& > img': { mr: 2, flexShrink: 0 } }}
+                              >
+                                <CryptoPairEntity symbol={option} />
+                              </Box>
+                            );
+                          }}
+                          {...field}
+                          onChange={(_, value) => field.onChange(value)}
+                        />
+                      );
+                    }}
+                  />
+                  {errors.pair && (
+                    <FormHelperText>{errors.pair.message}</FormHelperText>
                   )}
-                />
-                {errors.startDate && (
-                  <FormHelperText>{errors.startDate.message}</FormHelperText>
-                )}
-              </FormControl>
-              <FormControl error={!!errors.endDate} sx={{ width: '100%' }}>
-                <Controller
-                  name="endDate"
-                  control={control}
-                  render={({ field }) => (
-                    <DatePicker
-                      label="End Date"
-                      closeOnSelect
-                      disablePast
-                      {...field}
-                      value={dayjs(field.value)}
-                    />
+                </FormControl>
+              </Box>
+              <Box display="flex" gap={2}>
+                <FormControl error={!!errors.start_date} sx={{ width: '100%' }}>
+                  <Controller
+                    name="start_date"
+                    control={control}
+                    render={({ field }) => (
+                      <DatePicker
+                        label="Start Date"
+                        format="YYYY-MM-DD"
+                        closeOnSelect
+                        disablePast
+                        {...field}
+                        value={dayjs(field.value)}
```

Comment:
Just to clarify: in form it's going to pass a `Date` object that's is properly constructed for UTC because of `dayjs.extend(utc)`, right?

I.e. if I pick `2025-08-25` when my local timezone is GMT+3 - it should end up with `2025-08-25T00:00:00.000Z` in manifest and not `2025-08-24T21:00:00.000Z`

I haven't tried it live while creating escrow, but when tried in some local script it didn't give me proper UTC timestamp